LibCarla/source/carla/client/BlueprintLibrary.h

```diff
@@ -22,8 +22,7 @@ namespace client {
   /// @todo Works as a list but its actually a map. We should assess the use
   /// cases and reconsider this implementation.
   class BlueprintLibrary
-    : public EnableSharedFromThis<BlueprintLibrary>,
-      private NonCopyable {
+    : public EnableSharedFromThis<BlueprintLibrary> {
```

Comment:
Changed this due to a complaint about the move constructor, can look more into it.